simpref/simpref.go

```diff
@@ -0,0 +1,158 @@
+package simpref
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/bitrise-io/go-plist"
+	"github.com/bitrise-io/go-utils/v2/fileutil"
+	"github.com/bitrise-io/go-utils/v2/log"
+	"github.com/bitrise-io/go-utils/v2/pathutil"
+	"github.com/bitrise-io/go-xcode/v2/destination"
+)
+
+const (
+	DefaultIPhoneSimulatorPreferencesPth = "~/Library/Preferences/com.apple.iphonesimulator.plist"
+	defaultSimulatorDestination          = "platform=iOS Simulator,name=Bitrise iOS default,OS=latest"
+)
```

Comment:
These constants can be removed because they are not used.

Reply:
Removed in https://github.com/bitrise-steplib/steps-disable-iphonesimulator-connect-hardware-keyboard/pull/1/commits/efeb8e5d30852808f315fdaf935bcd52d2c4570c